AnalyzingAndProfilingTools/GameOfLife/GameOfLife/Grid.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Shapes;
+
+namespace GameOfLife
+{
+    class Grid

```

Comment:
please review this file and remove redundant code, also there are redundant loops and arrays, e.g. cells initialization in constructor

Reply:
Please check update comments commit